net/pfSense-pkg-pfBlockerNG-devel/files/usr/local/pkg/pfblockerng/pfblockerng_install.inc

```diff
@@ -159,9 +159,12 @@ if ($pfb['keep'] == 'on' && isset($pfb['widgets']) && strpos($pfb['widgets'], 'p
 // Create Firewall filter service and link required executables
 update_status("\nRemove any existing and create links for Firewall filter executables...");
 unlink_if_exists('/usr/local/bin/php_pfb');
-symlink('/usr/local/bin/php', '/usr/local/bin/php_pfb');
+if (file_exists('/usr/local/bin/php_pfb')) {
```

Comment:
`unlink_if_exists()` is called above, why add this `exec()` call here?

Reply:
Its a failsafe since some users are reporting that a `php_pfb -v` reports an older version of PHP.

It should have been deleted with the unlink() but that cmd might not do a force if its locked etc...